tests/test_proxy_use.py

```diff
@@ -77,26 +75,24 @@ def setUpClass(cls):
       raise NotImplementedError("TestWithProxies only works with Python 2"
                                 " (proxy_server.py is Python2 only)")
 
-    # Launch a simple HTTP server (serves files in the current dir).
     cls.http_port = random.randint(30000, 45000)
```

Comment:
Why not just let `TestServerProcess` generate this port number?

Reply:
It could, I just tried to make the lines shorter and just doesn't change it.
We are planning to leave all instances of port generation outside `tests/utils.py` and it's in the best interest of us if we do that as well.
I will change it.